app/controllers/users_controller.rb

```diff
@@ -0,0 +1,29 @@
+class UsersController < ApplicationController
+  include UsersHelper
+
+  def new; end
+
```

Comment:
- It should initialize a user, i.e. `@user = User.new`, then you can pass this to views or the form